charts/gardener-extension-provider-aws/templates/daemonset-custom-mtu.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       labels:
         k8s-app: mtu-customizer
     spec:
-      priorityClassName: gardener-extension-provider-aws
+      priorityClassName: gardener-system-90
```

Comment:
Is it `gardener-system-900` instead of `gardener-system-90`? Same for others below? ref: https://github.com/gardener/gardener/blob/master/docs/development/priority-classes.md

Reply:
Yes, its a typo thanks for noticing